packages/integration-tests/tests/users.test.ts

```diff
@@ -0,0 +1,88 @@
+import { User } from '@logto/schemas';
+
+import { authedAdminApi } from '@/api';
+import { generatePassword, generateUsername } from '@/utils';
+
+const createUser = async (username?: string) => {
+  const useUsername = username ?? generateUsername();
```

Comment:
1. This name is confusing
2. I didn't see the usage for assigned username...

Reply:
This is used in:

https://github.com/logto-io/logto/pull/1640/files#diff-2a55e5a4ae83380e5f03423dcb061b67bc3cc4a50be610327524085f51dd5864R21-R28

Reply:
per our discussion, instead of testing username, you should test user id.